extensions/ui/org.eclipse.smarthome.ui.basic/web-src/smarthome.js

```diff
@@ -1845,53 +1835,50 @@
 
 			function walkWidgets(widgets) {
 				widgets.forEach(function(widget) {
-					if (widget.item === undefined) {
+					if (widget.widgetId === undefined || smarthome.dataModel[widget.widgetId] === undefined) {
```

Comment:
linebreak between comparisons as line is too long

Reply:
Fixed